.travis.yml

```diff
@@ -12,8 +12,11 @@ matrix:
       env: TYPO3_VERSION="^8.7"
 
 env:
-  - TYPO3_VERSION="^7.6"
-  - TYPO3_VERSION="^8.7"
+  global:
+    - TYPO3_PATH_ROOT=$PWD/.Build/public
```

Comment:
not sure if this is still needed at all. maybe try completely removing it

Reply:
Lets see … #29

Reply:
Okay, it's actually not needed. Closing this PR then in favor of the new one.